src/FunderMaps.AspNetCore/FunderMapsStartup.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using System.Linq;
+using AutoMapper;
+using FunderMaps.AspNetCore.DataTransferObjects;
+using FunderMaps.AspNetCore.Extensions;
+using FunderMaps.Core.Entities;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+
+[assembly: HostingStartup(typeof(FunderMaps.AspNetCore.FunderMapsStartup))]
+
+namespace FunderMaps.AspNetCore
+{
+    public class FunderMapsStartup : IHostingStartup
+    {
+        private Core.AppContext AppContextFactory(IServiceProvider serviceProvider)
```

Comment:
Very elegant solution. I would suggest inverting the method order in this file for readability though.